src/asyncresults.jl

```diff
@@ -116,8 +114,8 @@ function _consume(jl_conn::Connection)
                     return result_ptrs
                 else
                     result_num = length(result_ptrs) + 1
-                    debug(LOGGER,
-                        "Saving result $result_num from connection $(jl_conn.conn)"
+                    debug(
+                        LOGGER, "Saving result $result_num from connection $(jl_conn.conn)",
```

Comment:
I thought we leave the trailing comma off when all args fit on one line?

Reply:
that seem arbitrary to me

Reply:
The trailing comma exists to make git diffs cleaner. When multiple args are on one line, the whole line will always change if the args change so the trailing comma serves no purpose. 

`black` does this for Python code.

Reply:
okay i am convinced 👍 Please can you comment over in https://github.com/domluna/JuliaFormatter.jl/issues/283 and (then i'll add it to the to-do list in the top comment)?

Reply:
even better https://github.com/domluna/JuliaFormatter.jl/pull/295#issuecomment-697763881